packages/atst/src/react.ts

```diff
@@ -0,0 +1,445 @@
+// Generated by @wagmi/cli@0.1.10 on 2/26/2023 at 11:08:05 AM
```

Comment:
Noice!
How do we keep this in sync with the version of wagmi we use?

Reply:
This is why I made wagmi a peer dependency.    The user of the atst has to install wagmi within the range we specified